src/manager/educational-institutes/edInstitutesManagerTypes.ts

```diff
@@ -38,9 +38,57 @@ export interface IFilters {
 }
 
 export interface IProvinces {
-  data: string[]
+  provinces: string[]
 }
 
 export interface EducationalInstitutesPageParam {
   page: number
 }
+
+export interface ICreateEducationalInstituteFormDataProps {
+  address: string
+  description: string
+  email: string
+  endTime: string
+  lecturerQuantity: string
+  name: string
+  phone: string
+  province: string
+  rector: string
+  startTime: string
+  studentQuantity: string
+  subtitle: string
+  type: string
+  [keys: number]: File
+}
+
+export interface ICreateEducationalInstituteProps {
+  address: string
+  description: string
+  email: string
+  endTime: string
+  lecturerQuantity: string | number
+  name: string
+  phone: string | number
+  imageURLs: ICreateEducationalInstituteImage[] | string[] | null
+  province: ICreateEducationalInstituteAutocompleteField
+  rector: string
+  startTime: string
+  studentQuantity: string | number
+  subtitle: string
+  type: ICreateEducationalInstituteAutocompleteField
+}
+
+export interface ICreateEducationalInstituteAutocompleteField {
+  id: number | string
+  name: string
+}
+
+export interface ICreateEducationalInstituteImage {
+  [key: number]: File
```

Comment:
Why key is number, if needed let's use array on other case key should be string

Reply:
key  is  index and index is number

Reply:
not sure what that means)

I say that if key is number we can use array if we need object there the key should be string